resources/domain/balance/index.ts

```diff
@@ -13,16 +21,16 @@ export function formatBalance (balance, totalValue, decimals = 8) {
   }).format(balance.toFixed(decimals, BigNumber.ROUND_FLOOR))
 }
 
-export function formatUsdRate (rate, decimals = 2) {
+export function formatUsdRate (rate:BigNumber, decimals = 2) {
   return rate.isNaN()
     ? UNKNOWN
     : new Intl.NumberFormat('us-US', {
         minimumFractionDigits: decimals,
         maximumFractionDigits: decimals
-      }).format(rate.toFixed(decimals, BigNumber.ROUND_FLOOR))
+      }).format(Number(rate.toFixed(decimals, BigNumber.ROUND_FLOOR)))
 }
 
-export function balance (rawBalance, quote = {}) {
+export function balance (rawBalance: Balance, quote:Rate['usd'] = {price: BigNumber(0), change24hr: BigNumber(0)}): DisplayedBalance {
```

Comment:
I don't think this second parameter is optional, looks like we always pass it in